frontend/pages/api/hello.ts

```diff
@@ -0,0 +1,13 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+type Data = {
+  name: string
+}
+
+export default function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
```

Comment:
dobrze jest też otypować co będzie zwracane, żeby później było łatwiej debbugować

Reply:
@IntiveMichal `Data` jest otypowane w 4 linijce, a `req` i `res` mają zimportowany typ od Next.js w linijce 1. To powinno wystarczyć?